### DISCUSSION TITLE
I couldn't open the app

### DISCUSSION THREAD

**Discussion by shaheem-pp** [fabiusBile/Spotify4BigSur#18] [Q&A] [answered]
After iinstallation in M1 Air(Monterey) shows a page -“SpotifyMain.app” can’t be opened because Apple cannot check it for malicious software.-

How to solve this?

**Comment by jenniverse** [comment, marked as answer]
System Preferences → Security & Privacy → General and click allow

**Comment by thomi100** [comment]
I think this could be added to the readme such that all people know that :)
But can it also be patched/fixed? 🤔
